node_modules/oae-core/createlink/bundles/zh_CN.properties

```diff
@@ -4,10 +4,10 @@ ADDING_LINK=\u6dfb\u52a0\u94fe\u63a5&quot;${linkName}&quot;
 ENTER_OR_PASTE_LINKS_HERE=\u5728\u8fd9\u91cc\u8f93\u5165\u6216\u7c98\u8d34\u94fe\u63a5(\u4e00\u884c\u4e00\u4e2a)
 LINK_ADD_FAILED=\u65e0\u6cd5\u6dfb\u52a0&quot;${linkName}&quot;\u94fe\u63a5
 LINK_ADD_SUCCESS=\u5df2\u6210\u529f\u6dfb\u52a0 &quot;${linkName}&quot;\u94fe\u63a5
-LINK_ADDED_TO_MY_LIBRARY_PLURAL=${count}\u4e2a\u94fe\u63a5\u5df2\u6dfb\u52a0\u5230 <a href\="/me/library"> \u6211\u7684\u56fe\u4e66\u9986</a>\u3002
-LINK_ADDED_TO_MY_LIBRARY_SINGULAR=&quot;<a href\="${link1URL}">${link1}</a>&quot;\u94fe\u63a5\u5df2\u88ab\u6dfb\u52a0\u5230 <a href\="/me/library"> \u6211\u7684\u56fe\u4e66\u9986</a>\u3002
-LINK_ADDED_TO_MY_LIBRARY_AND_CONTEXT_SINGULAR=\u94fe\u63a5&quot;<a href\="${link1URL}">${link1}</a>&quot;\u6dfb\u52a0\u5230<a href\="/me/library">\u6211\u7684\u56fe\u4e66\u9986</a>\u548c<a href\="${context1URL}">${context1}</a>\u3002
-LINK_ADDED_TO_MY_LIBRARY_AND_CONTEXT_PLURAL=${count}\u4e2a\u94fe\u63a5\u5df2\u6dfb\u52a0\u5230<a href\="/me/library">\u6211\u7684\u56fe\u4e66\u9986</a>\u548c<a href\="${context1URL}"><a href\="${context1URL}/library">${context1}</a>\u3002
+LINK_ADDED_TO_MY_LIBRARY_PLURAL=${count}\u4e2a\u94fe\u63a5\u5df2\u6dfb\u52a0\u5230 <a href\="/library"> \u6211\u7684\u56fe\u4e66\u9986</a>\u3002
+LINK_ADDED_TO_MY_LIBRARY_SINGULAR=&quot;<a href\="${link1URL}">${link1}</a>&quot;\u94fe\u63a5\u5df2\u88ab\u6dfb\u52a0\u5230 <a href\="/library"> \u6211\u7684\u56fe\u4e66\u9986</a>\u3002
```

Comment:
No, they don't need 2 spaces (the browser wouldn't render them anyway). As far as I know, this language is actually entirely space-less, so even the single space is probably a mistake.